### ISSUE TITLE
[BUG] This App is Not Work With Emulator or Rooted Device

### ISSUE THREAD

**Issue by viibhuGupta** [waydroid/waydroid#1094]
### Describe the bug

But This App Is works well when I Run This App With the Android Studio Emulator i Am not Getting Any Error Like This 
Please help me how i can solve this problem

### Waydroid version

01.30.2023

### Device

Linux Desktop

### Operating System

Fedora 38 

### Kernel version

6.4.15-200.fc38.x86_64 (64-bit)

### Desktop Environment

Kde

### GPU

Mesa Intel® Xe Graphics

### Logs

![photo_2023-09-16_13-01-32](https://github.com/waydroid/waydroid/assets/108029219/35503e4b-3d0f-4976-832d-07fa940e1abe)

**Comment by firefoxlover** [comment]
yes, what app? Add a link to the App and specifications, is it yours or why do you run it in Android Studio? Otherwise this issue is pure spam and should be closed

**Comment by Quackdoc** [comment]
can you try https://github.com/waydroid/waydroid/issues/1060#issuecomment-1732325955

**Comment by warlice** [comment]
It won't alert if  the build.product  not contains "sdk".
